src/main/kotlin/hs/kr/equus/user/domain/auth/entity/PassInfo.kt

```diff
@@ -0,0 +1,19 @@
+package hs.kr.equus.user.domain.auth.entity
+
+import org.springframework.data.annotation.Id
+import org.springframework.data.redis.core.RedisHash
+import org.springframework.data.redis.core.TimeToLive
+import org.springframework.data.redis.core.index.Indexed
+
+@RedisHash
+data class PassInfo(
+
+    @Id
+    val name: String,
+
+    @Indexed
+    val phoneNumber: String,
+
```

Comment:
🚫 **[ktlint]** <standard:no-blank-line-in-list> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unexpected blank line(s) in value parameter list